src/lib/Parser/DocblockVisitor.php

```diff
@@ -68,6 +68,23 @@ public function leaveNode(Node $node)
 
                         $line = sprintf('%s @%s %s%s', $match[1], $match[2], implode('|', $orTypes), $match[4]);
                     }
+
+                    $pattern = '/([{<>\\s@(\[\\\\"\'])(([a-zA-Z_][a-zA-Z0-9_]*((\\\\)+|))+)/m';
+
+                    preg_match_all($pattern, $line, $matches);
```

Comment:
Wouldn't it be safer to rely on `preg_match*` returned result before processing matches further? `if (false !== preg_match...) ...`, `if (false !== preg_match_all ...) ...`